### ISSUE TITLE
After app relaunch, SDK will not connect to product

### ISSUE THREAD

**Issue by charlieMonroe** [dji-sdk/Mobile-SDK-iOS#219]
When you manually quit the app on iOS (by double-clicking the home button and swiping up), or if it crashes, occasionally, the SDK fails to connect to the product. The app first registers with the SDK (successfully) and calls `DJISDKManager.startConnectionToProduct()` - which returns true, but the delegate method `func productConnected(_ product: DJIBaseProduct?)` never gets called.

The only solution is to unplug and re-plug the cable connecting the remote controller and the iPhone. After that I can see some logs from External Accessory framework and the device immediately connects. Is this a known issue?

DJI Phantom 4 Pro (Dec 25 firmware), remote controller up-to-date
iPhone 6
Xcode 9.2, iOS 11.2.5

**Comment by XietongLU** [comment]
Yes. It is a known issue of iOS 11. We are talking to Apple to resolve this issue.

**Comment by charlieMonroe** [comment]
Thanks for the quick confirmation!

**Comment by Nitaym** [comment]
From iOS 11.2.6 release notes:

> Fixes an issue where some third-party apps could fail to connect to external accessories

Did that fix the issue?

https://support.apple.com/en-us/HT208067

**Comment by charlieMonroe** [comment]
11.2.6 might have improved this, but hasn't fixed it completely - we're still able to reproduce this.

**Comment by XietongLU** [comment]
Confirmed with Apple, it is not fixed yet. We also found that the latest beta version of iOS may alleviate this issue.
